### PR TITLE
Missing info_block_show_title field

### PR DESCRIPTION
**Pull request by infograf768**
the info_block_show_title parameter displays in Protostar the sub title "Details" as coded in
https://github.com/joomla/joomla-cms/blob/staging/layouts/joomla/content/info_block/block.php#L21-L24

screen shot 2016-05-25 at 11 42 40

But the parameter was forgotten in the xmls, thus preventing to choose to display it or not.
This PR adds the parameter in all xmls concerned.
For example, in Articles Options, one will get the missing field with label "Details Title"

screen shot 2016-05-25 at 11 42 09

In order to get, if desired

screen shot 2016-05-25 at 11 46 50

### CODE REVIEW

**Comment by brianteeman** [comment]
Dont you think it should be called "Article Info Title"

On 25 May 2016 at 11:48, infograf768 <notifications@github.com> wrote:

> the info_block_show_title parameter displays in Protostar the sub title
> "Details" as coded in
>
> https://github.com/joomla/joomla-cms/blob/staging/layouts/joomla/content/info_block/block.php#L21-L24
>
> [image: screen shot 2016-05-25 at 11 42 40]
> <https://cloud.githubusercontent.com/assets/869724/15535486/dede21fe-226d-11e6-9e32-2ed99e636fba.png>
>
> But the parameter was forgotten in the xmls, thus preventing to choose to
> display it or not.
> This PR adds the parameter in all xmls concerned.
> For example, in Articles Options, one will get the missing field with
> label "Details Title"
>
> [image: screen shot 2016-05-25 at 11 42 09]
> <https://cloud.githubusercontent.com/assets/869724/15535529/1298456a-226e-11e6-9824-085b8baad094.png>
>
> In order to get, if desired
>
> [image: screen shot 2016-05-25 at 11 46 50]
> <https://cloud.githubusercontent.com/assets/869724/15535631/76c42e0a-226e-11e6-999f-d5a14ddee08a.png>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/joomla/joomla-cms/pull/10621
> Commit Summary
>
>    - Missing info_block_show_title field
>
> File Changes
>
>    - *M* administrator/components/com_content/config.xml
>    <https://github.com/joomla/joomla-cms/pull/10621/files#diff-0> (11)
>    - *M* administrator/components/com_content/models/forms/article.xml
>    <https://github.com/joomla/joomla-cms/pull/10621/files#diff-1> (11)
>    - *M* administrator/language/en-GB/en-GB.com_content.ini
>    <https://github.com/joomla/joomla-cms/pull/10621/files#diff-2> (2)
>    - *M* components/com_content/views/archive/tmpl/default.xml
>    <https://github.com/joomla/joomla-cms/pull/10621/files#diff-3> (11)
>    - *M* components/com_content/views/article/tmpl/default.xml
>    <https://github.com/joomla/joomla-cms/pull/10621/files#diff-4> (10)
>    - *M* components/com_content/views/category/tmpl/blog.xml
>    <https://github.com/joomla/joomla-cms/pull/10621/files#diff-5> (12)
>    - *M* components/com_content/views/featured/tmpl/default.xml
>    <https://github.com/joomla/joomla-cms/pull/10621/files#diff-6> (12)
>    - *M* layouts/joomla/content/info_block/block.php
>    <https://github.com/joomla/joomla-cms/pull/10621/files#diff-7> (2)
>
> Patch Links:
>
>    - https://github.com/joomla/joomla-cms/pull/10621.patch
>    - https://github.com/joomla/joomla-cms/pull/10621.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/10621>
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

File: administrator/language/en-GB/en-GB.com_content.ini [hunk 1]
```diff
@@ -64,6 +64,8 @@ COM_CONTENT_FIELD_IMAGE_CAPTION_LABEL="Caption"
 COM_CONTENT_FIELD_IMAGE_OPTIONS="Image Options"
 COM_CONTENT_FIELD_INFOBLOCK_POSITION_DESC="Puts the article information block above or below the text or splits it into two separate blocks, one above and the other below."
 COM_CONTENT_FIELD_INFOBLOCK_POSITION_LABEL="Position of Article Info"
+COM_CONTENT_FIELD_INFOBLOCK_TITLE_DESC="Displays the 'Details' title on top of the article information block."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_content.ini]: I think this should be 'Article Info' not 'details' 
it will make more sense this way especially with translations

**Comment by wojsmol** [comment]
@infograf768 Plase adjust new fields definitons to http://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

**Comment by infograf768** [comment]
@brianteeman 
Strings changed.

@wojsmol
The purpose of this PR is not to correct the formatting of all xmls concerned as they contain a mixture of different formats (useless imho to only correct the new field). I suggest, once this is merged, and if you wish, to make specific PRs as you did for other items already.

Please test PR. :smiley:

**Comment by brianteeman** [comment]
Thanks for changing the string. I agree with @wojsmol that you really should correct the xml code style at the same time

**Comment by infograf768** [comment]
There are 6 xmls with a LOT of fields in each. No time sorry.

**Comment by andrepereiradasilva** [comment]
@infograf768 i also agree with @wojsmol too. All new PR that use new xml should use the standard in those xml lines.

**Comment by brianteeman** [comment]
@infograf768 without fixing the codestyle then I am sorry but this isnt going to get  merged

**Comment by infograf768** [comment]
I don't mind. This was done to help. If someone volunteers to propose a PR towards my branch I will merge it.

**Comment by andrepereiradasilva** [comment]
we need some sort of automated travis cs for xml too. :wink:

**Comment by wojsmol** [comment]
@infograf768 I cen prepare PR against you branch :smile:

**Comment by brianteeman** [comment]
@wojsmol thank you

**Comment by wojsmol** [comment]
@infograf768 see https://github.com/infograf768/joomla-cms/pull/29 - only for new changes

**Comment by wojsmol** [comment]
@andrepereiradasilva PR fir cim_content is still in front of me.

**Comment by infograf768** [comment]
merged. thanks. but the other fields in these xmls are unchanged.

**Comment by wojsmol** [comment]
@infograf768 soon I create PR for com_content that enhances CS for XML and PHP.

**Comment by infograf768** [comment]
this still needs tests :smile: 
i suggest this to be merged first to avoid conflicts.

**Comment by andrepereiradasilva** [comment]
will test

File: layouts/joomla/content/info_block/block.php [hunk 2]
```diff
@@ -19,7 +19,7 @@
 				) : ?>
 
 			<dt class="article-info-term">
-				<?php if ($displayData['params']->get('info_block_show_title', 1)) : ?>
+				<?php if ($displayData['params']->get('info_block_show_title')) : ?>
```
Comment (andrepereiradasilva) [inline layouts/joomla/content/info_block/block.php]: why did you remove the default value of "Yes"?

**Comment by stellainformatica** [comment]
@test: patch tested, working fine

**Comment by stellainformatica** [comment]
I have tested this item :white_check_mark: successfully on b16dd66da7b28afb550b6675e20f8245a8c0bd4c<br /><br />Tested, working fine, thanks JM and wojsmol <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10621">issues.joomla.org/joomla-cms/10621</a>.</sub>

File: layouts/joomla/content/info_block/block.php (hunk 2 above)
Comment (infograf768) [inline layouts/joomla/content/info_block/block.php]: no use. see other params in same file
<?php if ($displayData['params']->get('show_parent_category')
etc.

Comment (andrepereiradasilva) [inline layouts/joomla/content/info_block/block.php]: i think, if you remove the default value, in a upgrade scenario the info block will now disapear until the user save the component settings.

**Comment by infograf768** [comment]
As I just added back the default, I have set both to successful test.


RTC. Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10621">issues.joomla.org/joomla-cms/10621</a>.</sub>

**Comment by roland-d** [comment]
Thanks everybody
